### PR TITLE
[aptos-framework] Clean up delegation_pool doc comments

### PR DESCRIPTION
**Pull request by stefanmendoza**
### <samp>🤖 Generated by Copilot at 6180652</samp>

This pull request fixes documentation errors, adds a new contributor, and updates website links. It improves the quality and accuracy of the `aptos-framework` and `developer-docs-site` documentation, and the usability of the `README.md` file.

### Test Plan

Here are two examples of the UI fixes.

#### Top of Page

**Before**
Screenshot 2023-05-02 at 1 37 51 PM

**After**
Screenshot 2023-05-02 at 2 37 29 PM

#### Example Flow

**Before**
Screenshot 2023-05-02 at 2 28 05 PM

**After**
Screenshot 2023-05-02 at 2 37 59 PM

### <samp>🤖 Generated by Copilot at 6180652</samp>

*  Remove asterisks and bullet points from introduction paragraph of `delegation_pool.md` ([link](https://github.com/aptos-labs/aptos-core/pull/8012/files?diff=unified&w=0#diff-b60ae3a7f7f83a3f278db0faf8ab33916117e83a5b4886b1ed71c3f3b6b17853L7-R90))
*  Fix typos and syntax errors in formulas and code snippets of `delegation_pool.md` ([link](https://github.com/aptos-labs/aptos-core/pull/8012/files?diff=unified&w=0#diff-b60ae3a7f7f83a3f278db0faf8ab33916117e83a5b4886b1ed71c3f3b6b17853L1034-R1034), [link](https://github.com/aptos-labs/aptos-core/pull/8012/files?diff=unified&w=0#diff-b60ae3a7f7f83a3f278db0faf8ab33916117e83a5b4886b1ed71c3f3b6b17853L1053-R1055), [link](https://github.com/aptos-labs/aptos-core/pull/8012/files?diff=unified&w=0#diff-b60ae3a7f7f83a3f278db0faf8ab33916117e83a5b4886b1ed71c3f3b6b17853L2225-R2225))
*  Add new contributor entry for `dhruvja` in `contributors.json` ([link](https://github.com/aptos-labs/aptos-core/pull/8012/files?diff=unified&w=0#diff-40a179e6c2032cda424adaff22fdd754f5f7da9481615d0bab453982a0254d5bR198-R202))
*  Update links to guide and tutorials pages in `README.md` ([link](https://github.com/aptos-labs/aptos-core/pull/8012/files?diff=unified&w=0#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L18-R19))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ Aptos is a layer 1 blockchain bringing a paradigm shift to Web3 through better t
 
 * [Aptos Labs](https://aptoslabs.com/)
 * [Aptos Developer Network](https://aptos.dev)
-* [Getting Started](https://aptos.dev/guides/getting-started)
```
Comment (stefanmendoza) [inline README.md:18]: This was a broken link.

File: developer-docs-site/src/components/MoveReference/index.tsx [hunk 2]
```diff
@@ -14,10 +14,10 @@ if (ExecutionEnvironment.canUseViewport) {
   window.addEventListener("hashchange", shiftWindow);
 }
 
-const root = "https://raw.githubusercontent.com/stefanmendoza/aptos-core/";
```
Comment (stefanmendoza) [inline developer-docs-site/src/components/MoveReference/index.tsx]: These changes are what I had to do to test my UI changes.

File: aptos-move/framework/aptos-framework/doc/delegation_pool.md [hunk 3]
```diff
@@ -4,89 +4,95 @@
...
+<li>Delegator A adds 100 stake which is converted to 100 shares into the active pool_u64</li>
+<li>Operator joins the validator set as the stake pool has now the minimum stake</li>
+<li>The stake pool earned rewards and now has 200 active stake. A's active shares are worth 200 coins as
+the commission fee is 0%.</li>
+<li></li>
+<ol>
+    <li>A requests <code>unlock</code> for 100 stake</li>
+    <li>Synchronization detects 200 - 100 active rewards which are entirely (0% commission) added to the active pool.</li>
+    <li>100 coins = (100 * 100) / 200 = 50 shares are redeemed from the active pool and exchanged for 100 shares
+into the pending_inactive one on A's behalf</li>
```
Comment (stefanmendoza) [inline aptos-move/framework/aptos-framework/doc/delegation_pool.md:71]: I guess Markdown doesn't support typed sublists (`a), b), c)` or `i., ii., iii.`), so added some HTML :/

**Comment by stefanmendoza** [comment]
@gregnazario I also noticed that this Markdown matches the comment at the top of the `delegation_pool` definition:
https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/delegation_pool.move#L1C1-L83

Is it auto-generated? Should we just update that comment to include what we have here? Or was it copied from that comment which is why we had all the asterisks?

File: aptos-move/framework/aptos-framework/doc/delegation_pool.md [hunk 4]
```diff
@@ -4,89 +4,95 @@
...
-* 12b. Synchronization detects no new inactive stake, but some pending_inactive rewards which are
-* distributed to the (2nd) pending_inactive pool
-* 12c. A's 50 shares = (50 * 100) / 50 = 100 coins are redeemed from the (1st) inactive pool and 100 stake
-* is transferred to A
-
-
+Allow multiple delegators to participate in the same stake pool in order to collect the minimum
+stake required to join the validator set. Delegators are rewarded out of the validator rewards
+proportionally to their stake and provided the same stake-management API as the stake pool owner.
+
```
Comment (gregnazario) [inline aptos-move/framework/aptos-framework/doc/delegation_pool.md:10]: This is generated from the code.  You'll have to change the comments in the move code for the module.

**Review by gregnazario — APPROVED** [review]

**Review by xindingw — APPROVED** [review]

**Pull request merged**